### PR TITLE
danna - add command

### PR DESCRIPTION
**Pull request by bossyshrimp68**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/lifter/neoLifter/NeoLifter.java [hunk 1]
```diff
@@ -16,7 +17,7 @@ public class NeoLifter implements ILifter {
     private TalonSRX solenoid;
 
     public NeoLifter() {
-        motor = new GBSparkMax(NeoLifterConstants.MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
```
Comment (Lisichker) [inline src/main/java/edu/greenblitz/robotName/subsystems/lifter/neoLifter/NeoLifter.java:19]: Nice.

File: src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.robotName.commands.arm;
+
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.wrist.WristConstants;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+public class MoveArmToIntakePosition extends ParallelCommandGroup {
+
```
Comment (Lisichker) [inline src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java:8]: I would have said something meaningful if I could understand any of this code

File: src/main/java/edu/greenblitz/robotName/commands/CollectNote.java [hunk 3]
```diff
@@ -11,9 +12,10 @@ public class CollectNote extends ConditionalCommand {
 
     public CollectNote() {
         super(
-                new NoteFromIntakeToShooter(),
```
Comment (Lisichker) [inline src/main/java/edu/greenblitz/robotName/commands/CollectNote.java:14]: It seems logical edit, but it feels too much to ask you for such a think as logic.

**Review by Lisichker — APPROVED** [review]

**Review by Lisichker — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/CollectNote.java [hunk 4]
```diff
@@ -11,9 +12,10 @@ public class CollectNote extends ConditionalCommand {
...
+                new MoveArmToIntakePosition().andThen(new NoteFromIntakeToShooter()),
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/CollectNote.java:15]: romy - i think its better if this is parrallel or race, its nice to have the arm roller hlping but i dont want it to take too much time. make sure tho that the arm default command or smth is this position, and then do a parrallel that runs the intake and arm roller

File: src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    public MoveArmToIntakePosition(){
+        super(
+                new MoveElbowAndWrist(ElbowConstants.PresetPositions.TRANSFER, WristConstants.PresetPositions.TRANSFER)
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java:11]:
```suggestion
                new MoveElbowAndWrist(
                     ElbowConstants.PresetPositions.TRANSFER, 
                     WristConstants.PresetPositions.TRANSFER
                )
```
btw dont commit suggestion (or do and then check) cus its stupid sometimes

**Review by greenblitz4590 — CHANGES_REQUESTED** [review]
romy - STOP OPENING PRS FROM MY ACCOUNT

File: src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java (hunk 2 above)
Comment (DANNA2828) [reply src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java:8]: thete is no need to be hurtful

File: src/main/java/edu/greenblitz/robotName/commands/CollectNote.java [hunk 6]
```diff
@@ -11,10 +12,13 @@ public class CollectNote extends ConditionalCommand {
 
     public CollectNote() {
         super(
-                new NoteFromIntakeToShooter(),
+                new NoteToIntake()
+                        .andThen(new MoveArmToIntakePosition())
+                        .alongWith(new NoteFromIntakeToShooter()),
                 new NoteToIntake()
```
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/commands/CollectNote.java:18]: you call this twice?

**Review by greenblitz4590 — CHANGES_REQUESTED** [review]

**Review by aaa-png — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/CollectNote.java (hunk 6 above)
Comment (YoniKiriaty) [reply src/main/java/edu/greenblitz/robotName/commands/CollectNote.java:18]: it is a conditional command, first call is if shooter is at angle, second if not

**Pull request closed without merging**